ui-tests/cypress/support/pageObjects/pages/designer/widgetconfigs/ContentQueryWidgetConfigPage.js

```diff
@@ -0,0 +1,18 @@
+import WidgetConfigPage from '../WidgetConfigPage';
+
+export default class ContentQueryWidgetConfigPage extends WidgetConfigPage {
+
+  getContentTypeField() {
+    return this.getMainContainer().find('[name="contentType"]');
+  }
+
+  getPublishSettingsAccordButton() {
+    return this.getMainContainer().contains(/^Publishing settings$/i);
```

Comment:
Avoid use contains to define the structure of a page

Reply:
specificity of this method has been expanded